prog/weave-kube/launch.sh

```diff
@@ -15,6 +15,12 @@ CONN_LIMIT=${CONN_LIMIT:-30}
 # Default for network policy
 EXPECT_NPC=${EXPECT_NPC:-1}
 
+# Ensure we have the required modules for NPC
+if [ "${EXPECT_NPC}" != "0" ]; then
+    modprobe br_netfilter
```

Comment:
I am seeing errors like:

`modprobe: module br_netfilter not found in modules.dep`

I guess this feature works only, if the module is **not compiled into** the kernel itself.
See a related discussion https://github.com/kubernetes/kubernetes/issues/12459